### PR TITLE
add: product page

### PR DESCRIPTION
**Pull request by bhushannaikaroskar**
Please review my PR for Product Listing Page
productpage

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -147,3 +157,71 @@ html {
...
+    margin: 1rem 0;
+}
+
+.grand-slider {
+    display: flex;
+    align-items: center;
+    appearance: none;
+    background-color: var(--PRIMARY-50);
+    height: 8px;
+    border-radius: 99px;
```
Comment (srishtigautam21) [inline style.css]: use rem here

**Review by srishtigautam21 — APPROVED** [review]
everything else seems fine...

**Review by saurabhan — APPROVED** [review]
looks good, approved

File: style.css (hunk 1 above)
Comment (bhushannaikaroskar) [reply style.css]: Thank you srishti for pointing this. Made required changes

**Pull request merged**